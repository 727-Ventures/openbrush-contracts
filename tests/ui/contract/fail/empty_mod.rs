#[openbrush::contract]
mod base_psp22 {
}

fn main() {}