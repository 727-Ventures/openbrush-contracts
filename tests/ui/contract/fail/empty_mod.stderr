error: missing ink! storage struct
 --> tests/ui/contract/fail/empty_mod.rs:2:1
  |
2 | mod base_psp22 {
  | ^^^
